scipy/stats/tests/test_distributions.py

```diff
@@ -1541,14 +1541,13 @@ def test_fit(self, rvs_shape, rvs_loc, rvs_scale, rng):
                      ndata / np.sum(np.log(data_shift/data_shift.min())))
         assert_equal(loc_mle_a, 2)
 
-    @pytest.mark.filterwarnings("ignore:invalid value encountered in "
-                                "double_scalars")
```

Comment:
The new warning here is encountered in "multiply".  I honestly think, I should just change it everywhere, `double_scalars` is pretty weird.  `multiply (scalar)` may make sense.

Reply:
If you have a minute, can you also check gh-16146, I think I'm seeing the same type of warnings (it is the same module) when backporting--do you suggest I backport this stuff?

Reply:
Yeah, backporting this can't hurt, since it generates a lot of warnings (I might yet tweak NumPy, but really we shouldn't be sensitive to such tweaks here I think).

All of the `RuntimeWarnings: invalid value...` there seem to be exactly this PR, yes.